test/pyro/test_pyro_distribution.py

```diff
@@ -0,0 +1,57 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import pyro.distributions as dist
```

Comment:
This test file is renamed to avoid confliction with `test/test_distribution.py`

Reply:
I believe a cleaner solution is to add a file `tests/pyro/__init__.py` so PyTest can distinguish the test module names. I just had this same problem a couple days ago in Pyro.

Reply:
Thanks!